.github/workflows/minimal-usage.yml

```
@@ -10,9 +13,15 @@ on:
 permissions: read-all
 jobs:
   minimal_usage:
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        include:
+          - os: ubuntu-24.04-arm
+          - os: ubuntu-24.04
+          - os: ubuntu-latest
```

I don't think you need to use `include` here, that's for adding specific combinations/configurations:
```suggestion
        os:
          - ubuntu-24.04-arm
          - ubuntu-24.04
          - ubuntu-latest
```